packages/core/test/application_init_spec.ts

```diff
@@ -73,5 +74,58 @@ import {inject, TestBed, waitForAsync} from '../testing';
            });
          })));
     });
+
+    describe('with app initializers represented using observables', () => {
+      let subscriber: Subscriber<any>;
+      let observable: Observable<any>;
+      let completerResolver = false;
```

Comment:
The `completerResolver` is used in existing Promise-based tests, but I think we can rename it in newly-added tests to be a bit more clear on what this flag stands for, for example:
```suggestion
      let initFnInvoked = false;
```

Reply:
I'd vote to change this in existing tests in the same file for consistency as well.